src/engraving/libmscore/edit.cpp

```diff
@@ -99,20 +101,6 @@ static ChordRest* chordOrRest(EngravingItem* el)
     return nullptr;
 }
 
-//---------------------------------------------------------
-//   getSelectedNote
-//---------------------------------------------------------
-
-Note* Score::getSelectedNote()
-{
-    EngravingItem* el = selection().element();
-    if (el && el->isNote()) {
-        return toNote(el);
-    }
-    MScore::setError(MsError::NO_NOTE_SELECTED);
```

Comment:
surely we can do better than using a static/global variable though...why not make it a proper member?